app/models/assignment.rb

```diff
@@ -10,4 +10,8 @@ def self.complete(describee_id, describer_id)
     assignment.update(completed?: true)
     assignment.save
   end
+
+  def self.valid?(describee_id, describer_id)
```

Comment:
@Dpalazzari @wlffann Should this method be `invalid?`? It seems to return true if it is empty.